src/test/resource/Logic_test.cpp

```diff
@@ -93,11 +94,11 @@ class ResourceManager_test : public beast::unit_test::suite
         beast::IP::Endpoint const addr (
             beast::IP::Endpoint::from_string ("192.0.2.2"));
 
-        using namespace std::placeholders;
-
         std::function<Consumer(beast::IP::Endpoint)> ep = limited ?
-            std::bind(&TestLogic::newInboundEndpoint, &logic, _1) :
-            std::bind(&TestLogic::newUnlimitedEndpoint, &logic, _1);
+            std::bind(
+                &TestLogic::newInboundEndpoint, &logic, std::placeholders::_1) :
+            std::bind(
+                &TestLogic::newUnlimitedEndpoint, &logic, std::placeholders::_1);
```

Comment:
What's the motivation for this change? I don't object, I just don't understand why it's needed to support the cmake unity build or removing the deprecated `target` variable.

Reply:
one of the compilers (lost track of which one) got confused between `_1` in a boost namespace and in stdlib for unity..obviously this happens now because different TUs are being grouped together that had not previously been.